packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.styles.ts

```diff
@@ -10,6 +12,9 @@ const GlobalClassNames = {
 export const getStyles = (props: IDetailsRowCheckStyleProps): IDetailsRowCheckStyles => {
   const { theme, className, isHeader, selected, anySelected, canSelect, compact, isVisible } = props;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
+  const { rowHeight, compactRowHeight } = DEFAULT_ROW_HEIGHTS;
+
+  const height = isHeader ? HEADER_HEIGHT : compact ? compactRowHeight : rowHeight;
```

Comment:
nit but this is a tad difficult to parse as-is, but it might be prettier to blame